docs/src/modules/components/DemoToolbar.js

```diff
@@ -532,36 +531,20 @@ export default function DemoToolbar(props) {
             {showCodeLabel}
           </Button>
           {demoOptions.hideEditButton ? null : (
-            <React.Fragment>
-              <DemoTooltip title={t('stackblitz')} placement="bottom">
-                <IconButton
-                  data-ga-event-category="demo"
-                  data-ga-event-label={demo.gaLabel}
-                  data-ga-event-action="stackblitz"
-                  onClick={() => stackBlitz.createReactApp(demoData).openSandbox()}
-                  {...getControlProps(4)}
-                  sx={{ borderRadius: 1 }}
-                >
-                  <SvgIcon viewBox="0 0 19 28">
-                    <path d="M8.13378 16.1087H0L14.8696 0L10.8662 11.1522L19 11.1522L4.13043 27.2609L8.13378 16.1087Z" />
-                  </SvgIcon>
-                </IconButton>
-              </DemoTooltip>
-              <DemoTooltip title={t('codesandbox')} placement="bottom">
-                <IconButton
```

Comment:
Why remove codesandbox?

Reply:
> Why remove codesandbox?

It's [broken](https://mui-org.slack.com/archives/C011VC970AW/p1744197923988119). And it doesn't look like they [prioritize](https://github.com/codesandbox/codesandbox-client/issues/8746) at all. Perhaps we could bring it back once their issues are fixed?

> It looks like the majority of our users go with codesandbox.

The numbers may also be a bit skewed as in some places in the docs there is only codesandbox available (e.g. joy templates)

Reply:
I brought back codesandbox, we can discuss removing it separately